src/components/CharacterForm/CharacterForm.js

```diff
@@ -54,17 +55,39 @@ const CharacterForm = ({ onClose }) => {
   const [errors, setErrors] = useState({})
   const { params } = router.query
 
+  const isEdit = useMemo(() => {
+    return params && params[0] === formModeEnum.edit
+  }, [params])
+
+  const character = useSelector((state) =>
+    isEdit ? getCharacterById(state, params[1]) : null
+  )
+
+  const defaultValues = {
+    name: character?.name || "",
+    eyeColor: character?.eyeColor || "",
+    height: character?.height || "",
```

Comment:
This is an inappropriate pattern, please read the react-hook-form docs on how to reset the form